generate/post_process_json_test.go

```diff
@@ -0,0 +1,29 @@
+package generate
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestJSONTagPostProcessor(t *testing.T) {
+	tests := []struct {
+		input  []byte
+		output []byte
```

Comment:
make these `string`, and then each case will be simpler - you have N cases, but only one loop body